branch-config.mk

```diff
@@ -19,8 +19,20 @@
 # conflicts on multiple places.
 
 
-# Name of the development branch. This could be master, fXX-release, rhelX-branch, rhel-X ...
-GIT_BRANCH ?= master
+# Name of the release branch. This could be master, fXX-release, rhelX-branch, rhel-X ...
+# For branched Fedora, this must be the -release branch.
+GIT_BRANCH ?= f34-release
 
 # Directory for this anaconda branch in anaconda-l10n repository. This could be master, fXX, rhel-8 etc.
-L10N_DIR ?= master
+L10N_DIR ?= f34
+
+# Base container for our containers.
+BASE_CONTAINER ?= registry.fedoraproject.org/fedora:34
+
+# Branch with spec file for dependency installation in container. This could be master, fXX-devel, rhel-X etc.
+# For branched Fedora, this must be the -devel branch
+CONTAINER_GIT_BRANCH ?= f34-devel
```

Comment:
This is a bit tricky. It should be `f34-release` for release branch to be correct because spec file is thing what can change specifically for release :(.

Reply:
Extended the comment according to discussion.